app/assets/javascripts/geoblacklight/modules/metadata.js

```diff
@@ -0,0 +1,16 @@
+
+var setRefUrl = function setRefUrl() {
```

Comment:
Certainly.  0447abe refactors this as `GeoBlacklight.MetadataDownloadButton`, and provides some basic `teaspoon` tests.